UI/CUIWnd.cpp

```diff
@@ -318,7 +318,10 @@ void CUIWnd::Render() {
         glDrawArrays(GL_LINE_LOOP,      m_buffer_indices[0].first, m_buffer_indices[0].second);
 
     } else {
-        glColor(ClientUI::WndColor());
+        if (m_flashing && GG::GUI::GetGUI()->Ticks() % (m_flash_duration * 2) > m_flash_duration)
+            glColor(GG::LightColor(ClientUI::WndColor()));
```

Comment:
Won't work well if the WndColor is already fairly light...

Reply:
Did some test with bright window background and I don't really think it is an issue. Did you have something specific in mind?

Reply:
Idea would be to make the colour darker instead of lighter if it's already more than ~70% white.

Reply:
Sounds okay to me, currently I'm hindered, I can add it later.